test/extended/util/image/image.go

```diff
@@ -38,14 +43,43 @@ func init() {
 		"registry.k8s.io/e2e-test-images/agnhost:2.43": 1,
 		"registry.k8s.io/e2e-test-images/nginx:1.15-4": 22,
 	}
+)
 
-	images = GetMappedImages(allowedImages, os.Getenv("KUBE_TEST_REPO"))
+func getImages() map[string]string {
+	initializationLock.RLock()
+	if !initialized {
+		fmt.Printf("Called getImages before initialization, starting wait.")
```

Comment:
Are we interested in figuring out where it's coming from? We could put [`runtime.Caller`](https://pkg.go.dev/runtime#Caller) info here